Tasks/Antipenko/Tasks/Task4/01.py

```diff
@@ -0,0 +1,42 @@
+'''
+1. Подготовиться к чтению содержимого файла text.txt
+2. Дать пользователю ввести с клавиатуры условный параметр "максимальное количество символов в строке",
+который должен быть больше 35
+3. Отформатировать текст с учётом максимального количества символов, при этом
+если слово целиком не умещается в строке, оно должно быть перенесено на следующую,
+а отступы между словами равномерно увеличены (по аналогии с функцией "Выровнять по ширине" текстовых редакторов)
+4. Записать получившийся текст в новый файл и оповестить об этом пользователя.
+(модуль textwrap использовать нельзя)
+'''
+
+while True:
+    am = input("Input the amount of characters per line(min - 36):\n")
+    if int(am) > 35 and am.isnumeric():
```

Comment:
have you tested it? it's a good shortcut, but you're using it wrong, for the short circuit to work properly it should be "am.isnumeric() and int(am) > 35", so the expression will fail in case the str is not numeric and will give you a number in case it is BUT there is one additional problem here - what if user inputs 0 - which resolves itself naturally in this case, because we have much bigger lower edge set, but still should be considered when implementing such expressions